app/Http/Transformers/Three/ReactionTransformer.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Rogue\Http\Transformers\Three;
+
+use Rogue\Models\Reaction;
+use League\Fractal\TransformerAbstract;
+
+class ReactionTransformer extends TransformerAbstract
+{
+    /**
+     * Transform the resource data.
+     *
+     * @param \Rogue\Models\Reaction $reaction
+     * @return array
+     */
+    public function transform(Reaction $reaction)
+    {
+        return [
+            'northstar_id' => $reaction->northstar_id,
+            'post_id' => (string) $reaction->post_id,
+            'term' => [
+                'name' => 'heart',
+                'id' => 641,
```

Comment:
what is the `641` id again?

Reply:
@sbsmith86 great q! I think that was the id for in P-A land for the heart "reaction" - should we cut this out since it's not applicable anymore?

Reply:
unless anything depends on it, I feel like we can kill it since for now we only have one reaction type.

Reply:
I also killed the `total` line under `term` bc I don't think it's relevant either! Not sure if we even want the `term` section now that I think of it?

Reply:
Yeah, we can remove all this old Phoenix stuff! 🔥

Reply:
@chloealee I don't think this was removed?

Reply:
@sbsmith86 oops I removed from the other `ReactionTransformer` by accident - thanks for flagging!